demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -335,6 +371,34 @@ def test_is_valid_rn(self, mocker, file_path, file_type):
         file_validator.validate_added_files(file_path, file_type)
         assert file_validator._is_valid
 
+    FILES_PATHS_FOR_ALL_VALIDATIONS = [
+        # ignoring images and change-logs
+        (DEFAULT_IMAGE, ''),
+        (VALID_MULTI_LINE_LIST_CHANGELOG_PATH, ''),
+        (INVALID_ONE_LINE_1_CHANGELOG_PATH, ''),
+        # validating files
+        (VALID_INTEGRATION_ID_PATH, 'integration'),
+        (VALID_TEST_PLAYBOOK_PATH, 'playbook'),
+        (VALID_SCRIPT_PATH, 'script'),
+        (VALID_DASHBOARD_PATH, 'dashboard'),
```

Comment:
what about beta integration?

Reply:
Added, although there is no beta-integration schema and the function `find_type` returns `integration` for beta integrations.